examples/api/lib/widgets/interactive_viewer/interactive_viewer.builder.0.dart

```diff
@@ -111,28 +67,20 @@ class _IVBuilderExampleState extends State<_IVBuilderExample> {
       child: LayoutBuilder(
         builder: (BuildContext context, BoxConstraints constraints) {
           return InteractiveViewer.builder(
-            alignPanAxis: true,
-            scaleEnabled: false,
-            transformationController: _transformationController,
+            boundaryMargin: const EdgeInsets.all(double.infinity),
             builder: (BuildContext context, Quad viewport) {
-              // A simple extension of Table that builds cells.
               return _TableBuilder(
-                  rowCount: 60,
-                  columnCount: 6,
                   cellWidth: _cellWidth,
+                  cellHeight: _cellHeight,
+                  viewport: axisAlignedBoundingBox(viewport),
                   builder: (BuildContext context, int row, int column) {
-                    if (!_isCellVisible(row, column, viewport)) {
-                      debugPrint('removing cell ($row, $column)');
-                      return Container(height: _cellHeight);
-                    }
-                    debugPrint('building cell ($row, $column)');
                     return Container(
                       height: _cellHeight,
+                      width: _cellWidth,
                       color: row % 2 + column % 2 == 1
                           ? Colors.white
                           : Colors.grey.withOpacity(0.1),
                       child: Align(
-                        alignment: Alignment.centerLeft,
                         child: Text('$row x $column'),
                       ),
                     );
```

Comment:
add a common between the "}" and the ")" on the next line and reformat.  It'll cascade nicely then (I know it was like that before, but might was well fix it).